.github/actions/find_latest_image/action.yaml

```diff
@@ -0,0 +1,50 @@
+name: Find latest OS image
+description: Finds the latest OS image of a given type.
+
+inputs:
+  imageType:
+    description: "Type of image to find. Can be one of [debug, release] or a custom prefix (branch name)."
+    required: true
+
+outputs:
+  image:
+    description: "The latest image of the given type."
+    value: ${{ steps.find-latest-image.outputs.image }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Login to AWS
+      uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838 # tag=v1.7.0
+      with:
+        role-to-assume: arn:aws:iam::795746500882:role/GithubConstellationImageFinder
+        aws-region: eu-central-1
```

Comment:
Which credentials are used to log in here? 
I see that we have an org secret registered, but my understanding is one has to pass secrets on to actions. 
I think we should be explicit here so as to not confuse which secret is used.

Reply:
All AWS accounts use the GitHub OpenID integration: https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services

There are no permanent secrets.